packages/@uppy/core/src/index.js

```diff
@@ -588,16 +586,18 @@ class Uppy {
   cancelAll () {
     this.emit('cancel-all')
 
-    // TODO Or should we just call removeFile on all files?
     const { currentUploads } = this.getState()
     const uploadIDs = Object.keys(currentUploads)
-
     uploadIDs.forEach((id) => {
       this._removeUpload(id)
     })
```

Comment:
I believe we can <strike>remove</strike>ditch the `_removeUpload` calls here; `removeFile` will remove each upload once all its files were removed.

(remove remove remove)

Reply:
Now the test is failing, I guess that’s ok, cause it’s creating uploads manually, with no files? So I should modify the test to add 2 files and then check both they and uploads are removed? https://github.com/transloadit/uppy/blob/bfd4166b93ff6cd3ef6431df6d47483e2387c088/packages/%40uppy/core/src/index.test.js#L243-L252